tests/Aserto.AspNetCore.Middleware.Tests/AsertoMiddlewareTests.cs

```diff
@@ -92,8 +92,7 @@ public async Task AsertoAuthorizationDisabledAllows()
                 o.ServiceUrl = "https://testserver.com";
                 o.AuthorizerApiKey = "YOUR_AUTHORIZER_API_KEY";
                 o.TenantID = "YOUR_TENANT_ID";
-                o.PolicyID = "YOUR_POLICY_ID";
-                o.PolicyRoot = "policy_root";
+                o.PolicyName = "YOUR_POLICY_NAME";
```

Comment:
Is the policy name needed here ?

Reply:
everything is mocked, not necessarily needed. But I think it's good to show that it can take it as input